documentation/3.4/content/release-notes.md

```diff
@@ -8,6 +8,7 @@ draft: false
 
 | Date               | Version  | Introduces backward incompatibilities?                                                                                                      | Change - See also, [Change log](#change-log).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 |--------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| February 17, 2023  | v3.4.6   | no                                                                                                                                          | Resolved several issues related to WDT models for Model in Image, resolved an issue related to an exception while correcting container port names, and updates including the WebLogic Monitoring Exporter.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
-> updates including the WebLogic Monitoring Exporter 
Every time I read this, I have a hard time discerning what you  mean, until I read the Change Log. What I would suggest is:
 -> updates including the WebLogic Monitoring Exporter version. OR
 -> updates including the version of WebLogic Monitoring Exporter.